README.md

```diff
@@ -132,12 +132,43 @@ currently, there are verification tasks for the ILP32 (32-bit) and the LP64 (64-
 
 In order to obtain verification tasks from the programs and specifications in the repository,
 a simple task-definition mechanism is used.
-For each program, the repository contains a .yml file that specifies 
-  - the subject files (the parts that a program consists of),
-  - the results (for each specification of the program, the expected outcome is indicated), and
-  - the parameters to be used for verification.
+For each program, the repository contains a .yml file that specifies the following items:
+  - `format_version`: the version of the format (a version string)
+  - `input_files`: the subject program files or directories
+    (a file or directory name, or a list of files or directory names, that the program consists of)
+  - `properties`: the properties that constitute the specification of the program,
+    each consisting of the following items:
+    - `property_file`: file that contains a property definition
+    - `expected_verdict`: the intended verification result (`true` or `false`)
+    - `subproperty` (optional): a subproperty of the property that is violated
+      in cases where the property is a conjunction of subproperties (for verdict `false`)
+  - `options`: parameters that are relevant for verification or give extra information:
+    - `language`: programming language that the program is written in (`C` or `Java`)
+    - `data_model` data model of the computer architecture
+      (`ILP32`, `LP64`, see http://www.unix.org/whitepapers/64bit.html)
   
-The [SV-COMP 2019 report] contains a description of the format with an example in Sect. 4 and Fig. 3.
+The [SV-COMP 2019 report] has documented the first version of the repository's task-definition format 1.0,
+and contains a description of the format with an example in Sect. 4 and Fig. 3.
+Format 1.1 adds the `options` dictionary.
+Here as example an extract of the task-definition file [c/list-properties/list-1.yml]:
+
+```
+format_version: '1.1'
+
+input_files: 'list-1.i'
+
+properties:
+  - property_file: ../properties/unreach-call.prp
+    expected_verdict: true
+  - property_file: ../properties/valid-memsafety.prp
+    expected_verdict: false
+    subproperty: valid-memtrack
+
+options:
+  language: C
+  data_model: ILP32
+```
+
```

Comment:
I miss a link to BenchExec for the general format description.

Reply:
That's a really good point.

I assumed that this repository (`sv-benchmarks`) defines the task-definition format,
but there's an [extensive example with defining comments](https://github.com/sosy-lab/benchexec/blob/master/doc/task-definition-example.yml) and [some technical details](https://github.com/sosy-lab/benchexec/blob/master/doc/benchexec.md#task-definition-files) in the BenchExec repository.

So at the moment, there's two documentations/definitions in two different places.

Wouldn't it be easier to maintain a single source of truth, i.e., only have a documentation in either BenchExec or sv-benchmarks repository, and link to that from the other repo?

Reply:
> Wouldn't it be easier to maintain a single source of truth, i.e., only have a documentation in either BenchExec or sv-benchmarks repository, and link to that from the other repo? 

I agree, I also would find this more natural. After all, If I want to know how the yml task definitions work, I would always search for documentation in `BenchExec` as a first guess.

The point that was made for doing it here was that `BenchExec` is a general purpose tool and does not need to know the specific conventions of SV-COMP. `BenchExec` supports the `options` in the yml-definitions and just passes them on. Their exact meaning is not defined by `BenchExec`. Other communities could introduce other options and that would work.

Nothing speaks against putting this README.md into the `contrib` folder at BenchExec or maybe `doc/contrib` or so. Then the `BenchExec` documentation could mention when explaining the `options` that there are domain-specific conventions in `doc/contrib` or so.

Reply:
> The point that was made for doing it here was that BenchExec is a general purpose tool and does not need to know the specific conventions of SV-COMP

Right. How about linking to the definition in the BenchExec repository and only defining the sv-benchmarks-specific `options` here?

For example:

> For each program, the repository contains a .yml file according to the [task-definition format defined by BenchExec](https://github.com/sosy-lab/benchexec/blob/master/doc/benchexec.md#task-definition-files).
In addition, task definitions use the following `options`-items specific to sv-benchmarks:
> - `language`: programming language that the program is written in (`C` or `Java`)
> - `data_model` data model of the computer architecture
>      (`ILP32`, `LP64`, see http://www.unix.org/whitepapers/64bit.html)
>
> [example with options-items from sv-benchmarks repo]

Reply:
> How about linking to the definition in the BenchExec repository and only defining the sv-benchmarks-specific `options` here?

I support this. The example could still be a full example.

Reply:
Great idea! Refactor out common code ^^.  

Just a minor remark: We should also specify here whether they are mandatory or not.

Reply:
> I miss a link to BenchExec for the general format description.

This repository contains the definition of the task-definition format. Thus, no link to BenchExec is necessary. But BenchExec should probably cite this repo as the source of the task-definition format. (It was developed for sv-benchmarks, not for BenchExec. BenchExec just supports our format.)

So, yes, this here is the main reference for the format. The full definition has to stay here, because there are (potentially?) other tool not only BenchExec that support our task-definition format.